Sources/Responses/BytesRepresentable.swift

```diff
@@ -1,3 +1,18 @@
+import Foundation
+
+// public extension Equatable where Self: BytesRepresentable {
+//   public static func == (lhs: Self, rhs: Self) -> Bool {
+//     return lhs.toBytes == rhs.toBytes
+//   }
+// }
+
 public protocol BytesRepresentable {
```

Comment:
My last attempt was to extend the Equality protocol to BytesRepresentable (which I didn't even know was possible), but that resulted in this error:
```
protocol 'BytesRepresentable' can only be used as a generic constraint because it has Self or associated type requirements
```
This might be the answer, but I haven't invested time in exploring this route because I'm not sure if changing all references to `BytesRepresentable` to generic types that conform to the protocol might break functionality.